src/edit/handler/Edit.Poly.js

```diff
@@ -80,6 +80,9 @@ L.Edit.Poly = L.Handler.extend({
     },
 
     _initMarkers: function () {
+
+        var maxPoints = this.options.maxPoints || 0;
```

Comment:
NIT: this can be defaulted to 0 by adding it to the options dict at the start of the class definition